app/res/layout/custom_alert_dialog.xml

```diff
@@ -83,7 +83,7 @@
 
                     android:layout_toLeftOf="@id/neutral_button"
 
-                    android:layout_marginRight="@dimen/standard_spacer"
+                    android:layout_marginRight="@dimen/standard_spacer_double"
```

Comment:
Hm, quick note that this is a broad-cutting change which could introduce UI changes which were unintentional. I'd be more comfortable pulling it with confirmation of whether dialogs with three elements look fine, still, since the neutral button will now have asymetric space between it and the other buttons